lib/chkname.c

```diff
@@ -56,6 +59,17 @@ login_name_max_size(void)
 static bool
 is_valid_name(const char *name)
 {
+	if (streq(name, "")
+	 || streq(name, ".")
+	 || streq(name, "..")
+	 || strpbrk(name, ",:")
+	 || strchriscntrl(name)
+	 || strisdigit(name))
+	{
+		errno = EINVAL;
+		return false;
+	}
+
 	if (allow_bad_names) {
 		return true;
 	}
```

Comment:
@zeha Do we have an agreement here?  :-)

Reply:
Yeah that seems like a good plan to me.

Reply:
I'll take that as an Acked-by, if you don't mind.  Thanks!

Reply:
Yes.

Acked-by: Chris Hofstaedtler <zeha@debian.org>